index.bs

```diff
@@ -281,6 +281,30 @@ In these cases, features should be designed such that it is possible for
 code to gracefully fail or be polyfilled.
 See [[#feature-detect]].
 
+<h3 id=new-features>New features can add functionality to the Web</h3>
+
+A new Web feature can add capabilities to the Web with little concern about the effect on existing content.
+The Web includes many extension points that allow for additions;
+see for example [[HTML#extensibility]].
+
```

Comment:
Possible addition:

> Before adding items, consider integration with existing, similar capabilities.  If this leads to a preferred design approach that cannot be implemented by only adding items, it might still be possible; see [[#removing-features]].

Reply:
Agreed, but it needs to be higher up (at least the first sentence)

Reply:
```suggestion

Before adding items, consider integration with existing, similar capabilities.
If this leads to a preferred design approach that cannot be implemented by only adding items,
it might still be possible; see [[#removing-features]].
Do not assume that a change or removal is impossible without first checking.
```

Reply:
> Before adding items, consider integration with existing, similar capabilities.

This could just go to the top.